Source/EbsdLib/EbsdHeaderEntry.h

```diff
@@ -110,8 +110,11 @@
 
 
     private:
+    public:
```

Comment:
`private: public:` should probably just be `public:`
Consider deleting one of the empty lines above this as well.